docs/getting_started/getting_started.rst

```diff
@@ -66,7 +62,7 @@ OpenHatch holds development meetings on IRC; our goal is to hold these meetings
 
 
 Read more documentation
-=======================
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
The Hitchhikers Guide to Python by Kenneth Reitz and team has a good style guide for rST docs. I plan to incorporate with attribution or use it as a guide to an OpenHatch rST style guide.